apps/console/src/features/applications/components/settings/sign-on-methods/script-based-flow/script-based-flow.tsx

```diff
@@ -144,9 +150,109 @@ export const ScriptBasedFlow: FunctionComponent<AdaptiveScriptsPropsInterface> =
     const [ showScriptResetWarning, setShowScriptResetWarning ] = useState<boolean>(false);
     const [ showConditionalAuthContent, setShowConditionalAuthContent ] = useState<boolean>(isMinimized);
     const [ isEditorFullScreen, setIsEditorFullScreen ] = useState<boolean>(false);
+    const [ showAddSecretModal, setShowAddSecretModal ] = useState<boolean>(false);
+    const [ editorInstance, setEditorInstance ] = useState(undefined);
+
+    /**
+     * List of secrets for the selected {@code secretType}. It can hold secrets of
+     * either a custom one or the static type "ADAPTIVE_AUTH_CALL_CHOREO"
+     */
+    const [ secretList, setSecretList ] = useState<SecretModel[]>([]);
+    /**
+     * List of secrets for facilitating secret list search function.
+     */
+    const [ filteredSecretList, setFilteredSecretList ] = useState<SecretModel[]>([]);
+    const [ isSecretListLoading, setIsSecretListLoading ] = useState<boolean>(true);
 
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    /**
+     * Calls method to load secrets to secret list.
+     */
+    useEffect(() => {
+        loadSecretListForSecretType();
+    }, []);
+
+    /**
+     * Loads secret list for adaptive auth secrets.
+     */
+    const loadSecretListForSecretType = async (): Promise<void> => {
```

Comment:
Do we need async here, since there's no `await` inside?
```suggestion
    const loadSecretListForSecretType = async (): Promise<void> => {
```

Reply:
Fixed